### ISSUE TITLE
New and revamped MTR-W core peripheral signals!

### ISSUE THREAD

**Issue by jpaulnz** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#10296]
_**[This page still reflects the old signal names example, Display which has now changed to "Front of Room" Attached screenshot for reference**_
![new signal name example](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/assets/101783267/18da756c-c61a-4562-b9cf-5d71e57d65d0)
![Old signal names](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/assets/101783267/0bab72b1-db5a-4161-aee4-b965cf1b6ac0)
]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b01dff81-f785-6949-4660-50f59f107f85
* Version Independent ID: b01dff81-f785-6949-4660-50f59f107f85
* Content: [Microsoft Teams Rooms Pro Management Portal - Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/rooms/managed-meeting-rooms-portal)
* Content Source: [Teams/rooms/managed-meeting-rooms-portal.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/rooms/managed-meeting-rooms-portal.md)
* Service: **msteams**
* Sub-service: **itpro-rooms**
* GitHub Login: @altsou
* Microsoft Alias: **altsou**

**Comment by get-itips** [comment]
Hello @jpaulnz 
Thanks for your feedback.
Couple of questions, is the first screenshot from the MTR Pro portal? And I see you also highlighted Conference mic and Camera, did you recognize those also changed?

Thank you

**Comment by jpaulnz** [comment]
@get-itips  Yes the first screenshot is from the MTR Pro Portal. 

RE : Conference Microphone, Conference
![Conferencing Microphone](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/assets/101783267/87ecb9d6-3c54-4b71-a7cf-22fbbde1c505)
![Conferencing Speaker](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/assets/101783267/7546c078-2b3f-4115-868f-2754bcdbc1f9)
![Room Camera](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/assets/101783267/b1add133-bf35-41a0-8874-35c0e93098a1)
 Speaker and Camera, yes as per the MTR Pro portal, they are also changed. Attaching screenshots from the portal for the change of Signals.

**Comment by get-itips** [comment]
Thanks for this while I try to get access to a MTR Portal...

And it seems that the Severity also changed from Important to Critical...

**Comment by jpaulnz** [comment]
@get-itips  Yes, It would be great if you could please go through the updates section on the MTR Pro Portal and make relevant changes to the official Microsoft Docs on the website. Thanks

**Comment by jpaulnz** [comment]
@get-itips  Hi. Has the changes been applied as I can still see the old names in the webpage ? https://learn.microsoft.com/en-us/microsoftteams/rooms/managed-meeting-rooms-portal

**Comment by get-itips** [comment]
The PR is not yet merged @jpaulnz
